qubesmanager/qube_manager.py

```diff
@@ -1398,10 +1400,16 @@ def update_network_menu(self):
     # noinspection PyArgumentList
     @pyqtSlot(name='on_action_createvm_triggered')
     def action_createvm_triggered(self):
-        with common_threads.busy_cursor():
-            create_window = create_new_vm.NewVmDlg(
+        if self.new_qube.state() == QProcess.ProcessState.Running:
+            return
+        if path.isfile("/usr/bin/qubes-new-qube"):
+            self.new_qube.start("/usr/bin/qubes-new-qube")
+        else:
+            # for stubborn users who might uninstall `desktop-linux-manager`
```

Comment:
Given how much functionality is there, I think that if a user manually uninstalls desktop-linux-manager we can safely leave them on their own.... Same as with qubes-global-config, if a user uninstalls a system package manually, some things will indeed fail. I think we can just remove the old create_new_vm dialog in its entirely and call here the `qubes-new-qube` dialog.

Reply:
OK. So lets go for the kill and delete `create_new_vm.py` and the related ui file. I will readjust the PR.

Reply:
OK. This is done properly. The main executable, desktop file, unittest, ui file are deleted. Packaging files, project files, translations, Makefile and setup file are updated.